services/api_service/api_service/urls.py

```diff
@@ -0,0 +1,23 @@
+"""api_service URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+from .views import index
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
```

Comment:
Do we need a Django admin here?

Reply:
If we will use user model for the rest api, then yes. Otherwise, no

Reply:
we can remove this for now then.